libtransmission/rpc-server.c

```diff
@@ -560,12 +560,31 @@ static void handle_request(struct evhttp_request* req, void* arg)
 
     if (req != NULL && req->evcon != NULL)
     {
+        static int attempts = 0;
         char const* auth;
         char* user = NULL;
         char* pass = NULL;
 
         evhttp_add_header(req->output_headers, "Server", MY_REALM);
 
+        if (attempts == 100)
+        {
+            send_simple_response (req, 403,
+                "Too many unsuccessful login attempts. "
+                "Please restart transmission-daemon.");
```

Comment:
There is no point in wrapping the line if it fits in current length limit (which is 128 as specified in uncrustify.cfg) and there is no particular structure to it (like HTML paragraphs below). Same below on `unauthuser` initialization.